src/Lib/LanguagesLib.php

```diff
@@ -183,6 +183,7 @@ public static function get_Iso639_3_To_Iso639_1_Map() {
             'sag' => 'sg', // Sango
             'san' => 'sa', // Sanskrit
             'sin' => 'si', // Sinhala
+            'skr' => 'sr', // Saraiki
```

Comment:
`sr` is the ISO 639-1 code for Serbian (you can find it under `srp` below). I don't think there's an ISO 639-1 code for Saraiki, so this change will have to be reverted.

Reply:
Done, thanks for checking for me!

Reply:
Is anything else still a problem?

Reply:
Language is not added in site yet.

Reply:
@mpQadir The language will be added on the website when this pull request and merged and when the changes are deployed on production. For now this pull request is still open.

@DJ-Saidez I think everything's good now :)

Reply:
I'll merge this now and deploy on Sunday. @DJ-Saidez Thank you for your work!

Reply:
This PR is now live. 🎉